doc/examples/effects/simple/lib/main_move.dart

```diff
@@ -17,9 +18,19 @@ class MyGame extends BaseGame with TapDetector {
   @override
   void onTapUp(details) {
```

Comment:
nit: Offset details

Reply:
or rather Type details (not sure what it is)

Reply:
Fixed